Makefile

```diff
@@ -1,6 +1,17 @@
+BUILDDIR = bin/livelint
+BUILDVERSION := $(or $(shell git describe --abbrev=0 --tags),development)
+BUILDDATE := $(shell date +%Y-%m-%d\ %H:%M)
+GITHASH := $(shell git rev-list -1 HEAD)
+
+LDFLAGS=-ldflags="-w -s -X 'main.buildversion=${BUILDVERSION}' -X 'main.builddate=${BUILDDATE}' -X 'main.githash=${GITHASH}'"
+
 .PHONY: build
 all:
-	go build -o bin/livelint
+	go build -o ${BUILDDIR}
+
+.PHONY: build_ldflags
```

Comment:
This is incomplete (the `.PHONY` statement always needs to be the same as its following command).

I would not make this a new `make` command, but rather just change the `all` one to always include the build flags.

Reply:
ok let's leave everything in one step